change/@fluentui-react-b7fd6bb1-a3d9-4b38-a59a-df2f423c337c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 16 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26805&commitId=ddf726d85fe0d1628adc46b108c9c4b5ee6ce78e&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1031 screenshots unchanged, 0 screenshots with different dimensions and 12 screenshots with visible difference.


  <details><summary><b>unknown</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Primary Anchor.default.chromium.png|94| Changed|
|Button (compat).Primary Anchor.hover.chromium.png|96| Changed|
|Button (compat).Primary Anchor.pressed.chromium.png|108| Changed|
|Button Special Scenarios (compat).no flex shrink.default.chromium.png|4719| Changed|
|MessageBar.Root actions - RTL.default.chromium.png|2824| Changed|
|MessageBar.Root actions single line - RTL.default.chromium.png|334| Changed|
|MessageBar.Root actions single line.default.chromium.png|334| Changed|
|MessageBar.Root actions.default.chromium.png|2824| Changed|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|3058| Changed|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|334| Changed|
|MessageBar.Root dismiss and action single line.default.chromium.png|334| Changed|
|MessageBar.Root dismiss and action.default.chromium.png|3058| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>